doc/proposals/tls-utilities.md

```diff
@@ -1,18 +1,30 @@
 # TLS Utilities proposal
 
-## Background:
+Implementation Owner:
+
+> Status: **deferred**
```

Comment:
Technically, this is implemented, and I don't think we've made any statement yet about its deprecation. Let's just go with `implemented` for now

```suggestion
> Status: **implemented**
```